File: src/main/java/com/ksptool/ql/biz/model/vo/GetModelRoleListVo.java
package com.ksptool.ql.biz.model.vo;

import lombok.Data;

@Data
public class GetModelRoleListVo {
    // 角色ID
    private Long id;
    
    // 角色名称
    private String name;
    
    // 头像路径
    private String avatarPath;
}

File: src/main/resources/views/aether/src/entity/model-role/vo/GetModelRoleListVo.ts
/**
 * 获取模型角色列表项VO
 */
export default interface GetModelRoleListVo {
    /**
     * 角色ID
     */
    id: string;
    
    /**
     * 角色名称
     */
    name: string;
    
    /**
     * 头像路径
     */
    avatarPath?: string;
} 